packages/components/src/ui/card/README.md

```diff
@@ -13,7 +13,8 @@ import {
 	CardBody,
 	CardFooter,
 	Text,
-} from '@wordpress/components/ui';
+	Heading,
```

Comment:
It's exposed in documentation as `__experimentalHeading`, so I guess it should be aliased here.